### ISSUE TITLE
VS Code Output panel broken in latest versions 

### ISSUE THREAD

**Issue by JoshWhite** [wallabyjs/public#2505]
### Leaving Output panel causes it to break

Recently, the output panel hasn't been working correctly in VS Code (see screen capture below). Basically, when switching away from the panel, the colours & links are lost. 

I've tried installing previous versions of Wallaby, but I get the same issue, so this may be an issue with the latest VS Code build. 

I've also tried this on multiple projects and get the same result, so I don't think it can be anything to do with my config. I've included the diagnostics report anyway, in case the OS version etc helps. 

Let me know if you need any more info 👍 

### Screen capture
![wallaby-output-panel](https://user-images.githubusercontent.com/1931725/91546052-c9f5ba00-e919-11ea-8967-498ca5daf5ab.gif)

### Wallaby diagnostics report

```
{
  editorVersion: '1.48.2',
  pluginVersion: '1.0.234',
  editorType: 'VSCode',
  osVersion: 'win32 10.0.19041',
  nodeVersion: 'v12.9.0',
  coreVersion: '1.0.946',
  checksum: 'MmQ1OWMxNjk0MGJhOWI1YTRlYWI2MDg4OTVhOGE1NTUsMTYyMDg2NDAwMDAwMCww',
  config: {
    diagnostics: {
      jest: {
        config: {
          configs: [
            {
              automock: false,
              browser: false,
              cache: true,
              cacheDirectory: 'C:\\Users\\josh\\AppData\\Local\\Temp\\jest',
              clearMocks: false,
              coveragePathIgnorePatterns: [ '\\\\node_modules\\\\' ],
              cwd: 'C:\\personal\\air-monitor-reporting',
              dependencyExtractor: null,
              detectLeaks: undefined,
              detectOpenHandles: undefined,
              displayName: undefined,
              errorOnDeprecated: false,
              extraGlobals: undefined,
              filter: null,
              forceCoverageMatch: [],
              globalSetup: null,
              globalTeardown: null,
              globals: {},
              haste: { computeSha1: false, providesModuleNodeModules: [], throwOnModuleCollision: false },
              moduleDirectories: [ 'node_modules' ],
              moduleFileExtensions: [
                'web.js',  'js',
                'web.ts',  'ts',
                'web.tsx', 'tsx',
                'json',    'web.jsx',
                'jsx',     'node'
              ],
              moduleLoader: undefined,
              moduleNameMapper: [ [ '^react-native$', 'react-native-web' ], [ '^.+\\.module\\.(css|sass|scss)$', 'identity-obj-proxy' ] ],
              modulePathIgnorePatterns: [],
              modulePaths: [],
              name: 'c09f621c9bf7fdb7510190bf7cb18de2',
              prettierPath: 'prettier',
              resetMocks: false,
              resetModules: false,
              resolver: null,
              restoreMocks: false,
              rootDir: 'C:\\personal\\air-monitor-reporting',
              roots: [ 'C:\\personal\\air-monitor-reporting' ],
              runner: 'jest-runner',
              setupFiles: [ 'C:\\personal\\air-monitor-reporting\\node_modules\\react-app-polyfill\\jsdom.js' ],
              setupFilesAfterEnv: [],
              skipFilter: false,
              skipNodeResolution: undefined,
              snapshotResolver: undefined,
              snapshotSerializers: [],
              testEnvironment: 'C:\\personal\\air-monitor-reporting\\node_modules\\jest-environment-jsdom-fourteen\\lib\\index.js',
              testEnvironmentOptions: {},
              testLocationInResults: false,
              testMatch: [ 'C:/personal/air-monitor-reporting/src/**/__tests__/**/*.{js,jsx,ts,tsx}', 'C:/personal/air-monitor-reporting/src/**/*.{spec,test}.{js,jsx,ts,tsx}' ],
              testPathIgnorePatterns: [ '\\\\node_modules\\\\' ],
              testRegex: [],
              testRunner: 'C:\\personal\\air-monitor-reporting\\node_modules\\jest-jasmine2\\build\\index.js',
              testURL: 'http://localhost',
              timers: 'real',
              transform: [
                [ '^.+\\.(js|jsx|ts|tsx)$', 'C:\\personal\\air-monitor-reporting\\node_modules\\react-scripts\\config\\jest\\babelTransform.js' ],
                [ '^.+\\.css$', 'C:\\personal\\air-monitor-reporting\\node_modules\\react-scripts\\config\\jest\\cssTransform.js' ],
                [ '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)', 'C:\\personal\\air-monitor-reporting\\node_modules\\react-scripts\\config\\jest\\fileTransform.js' ]
              ],
              transformIgnorePatterns: [ '[\\\\\\\\]node_modules[\\\\\\\\].+\\.(js|jsx|ts|tsx)$', '^.+\\.module\\.(css|sass|scss)$' ],
              unmockedModulePathPatterns: undefined,
              watchPathIgnorePatterns: []
            }
          ],
          globalConfig: {
            bail: 0,
            changedFilesWithAncestor: false,
            changedSince: undefined,
            collectCoverage: false,
            collectCoverageFrom: [ 'src/**/*.{js,jsx,ts,tsx}', '!src/**/*.d.ts' ],
            collectCoverageOnlyFrom: undefined,
            coverageDirectory: 'C:\\personal\\air-monitor-reporting\\coverage',
            coverageReporters: [ 'json', 'text', 'lcov', 'clover' ],
            coverageThreshold: null,
            detectLeaks: undefined,
            detectOpenHandles: undefined,
            enabledTestsMap: undefined,
            errorOnDeprecated: false,
            expand: false,
            extraGlobals: undefined,
            filter: null,
            findRelatedTests: undefined,
            forceExit: undefined,
            globalSetup: null,
            globalTeardown: null,
            json: false,
            lastCommit: undefined,
            listTests: undefined,
            logHeapUsage: undefined,
            maxConcurrency: 5,
            maxWorkers: 7,
            noSCM: undefined,
            noStackTrace: false,
            nonFlagArgs: undefined,
            notify: false,
            notifyMode: 'failure-change',
            onlyChanged: undefined,
            onlyFailures: undefined,
            outputFile: undefined,
            passWithNoTests: undefined,
            projects: null,
            replname: undefined,
            reporters: undefined,
            rootDir: 'C:\\personal\\air-monitor-reporting',
            runTestsByPath: false,
            silent: undefined,
            skipFilter: false,
            testFailureExitCode: 1,
            testNamePattern: undefined,
            testPathPattern: '',
            testResultsProcessor: null,
            testSequencer: 'C:\\personal\\air-monitor-reporting\\node_modules\\@jest\\test-sequencer\\build\\index.js',
            testTimeout: undefined,
            updateSnapshot: 'new',
            useStderr: false,
            verbose: null,
            watch: false,
            watchAll: undefined,
            watchPlugins: [
              { config: {}, path: 'C:\\personal\\air-monitor-reporting\\node_modules\\jest-watch-typeahead\\filename.js' },
              { config: {}, path: 'C:\\personal\\air-monitor-reporting\\node_modules\\jest-watch-typeahead\\testname.js' }
            ],
            watchman: true
          },
          hasDeprecationWarnings: false,
          wallaby: {
            roots: [],
            watchPathIgnorePatterns: [ '\\\\node_modules\\\\', '\\.\\\\dist\\\\|\\.\\\\build\\\\|\\.\\\\coverage\\\\|\\\\\\..+\\\\' ],
            testPathIgnorePatterns: [ '\\\\node_modules\\\\', '\\.\\\\dist\\\\|\\.\\\\build\\\\|\\.\\\\coverage\\\\|\\\\\\..+\\\\' ],
            testMatch: [ 'src/**/__tests__/**/*.{js,jsx,ts,tsx}', 'src/**/*.{spec,test}.{js,jsx,ts,tsx}' ],
            testRegex: []
          }
        }
      }
    },
    testFramework: { version: 'jest@24.8.0', configurator: 'jest@24.8.0', reporter: 'jest@24.8.0', starter: 'jest@24.8.0', autoDetected: true },
    filesWithCoverageCalculated: [ 'src/**/*.{js,jsx,ts,tsx}', '!src/**/*.d.ts' ],
    globalSetup: false,
    micromatch: true,
    files: [
      { pattern: '\\\\node_modules\\\\', regexp: /\\node_modules\\/, ignore: true, trigger: true, load: true },
      { pattern: '\\.\\\\dist\\\\|\\.\\\\build\\\\|\\.\\\\coverage\\\\|\\\\\\..+\\\\', regexp: /\.\\dist\\|\.\\build\\|\.\\coverage\\|\\\..+\\/, ignore: true, trigger: true, load: true },
      { pattern: '**/**', ignore: false, trigger: true, load: true, order: 1 },
      { pattern: 'src/**/__tests__/**/*.{js,jsx,ts,tsx}', ignore: true, trigger: true, load: true },
      { pattern: 'src/**/*.{spec,test}.{js,jsx,ts,tsx}', ignore: true, trigger: true, load: true }
    ],
    tests: [
      { pattern: '\\\\node_modules\\\\', regexp: /\\node_modules\\/, ignore: true, trigger: true, load: true, test: true },
      {
        pattern: '\\.\\\\dist\\\\|\\.\\\\build\\\\|\\.\\\\coverage\\\\|\\\\\\..+\\\\',
        regexp: /\.\\dist\\|\.\\build\\|\.\\coverage\\|\\\..+\\/,
        ignore: true,
        trigger: true,
        load: true,
        test: true
      },
      { pattern: 'src/**/__tests__/**/*.{js,jsx,ts,tsx}', ignore: false, trigger: true, load: true, test: true, order: 2 },
      { pattern: 'src/**/*.{spec,test}.{js,jsx,ts,tsx}', ignore: false, trigger: true, load: true, test: true, order: 3 }
    ],
    filesWithNoCoverageCalculated: [],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    compilers: {},
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    extensions: {},
    env: { type: 'node', params: {}, runner: 'C:\\Program Files\\nodejs\\node.exe', viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: true,
    configCode: 'auto.detect#-1393150686'
  },
  packageJSON: {
    dependencies: {
      '@chakra-ui/core': '^0.8.0',
      '@emotion/core': '^10.0.34',
      '@emotion/styled': '^10.0.27',
      '@metamodules/postgres': '^1.2.1',
      '@reach/router': '^1.3.4',
      '@reecelucas/react-datepicker': '^2.0.0',
      'decimal.js': '^10.2.0',
      'deploy-node-app': '^1.4.3',
      dotenv: '^8.0.0',
      'emotion-theming': '^10.0.27',
      formik: '^2.1.5',
      i: '^0.3.6',
      install: '^0.13.0',
      'node-sass': '^4.14.1',
      npm: '^6.14.8',
      ramda: '^0.27.1',
      react: '^16.8.6',
      'react-dom': '^16.8.6',
      'react-scripts': '3.0.1',
      'react-time-picker': '^4.0.1',
      'styled-components': '^5.1.1'
    },
    devDependencies: {
      'babel-eslint': '10.0.1',
      concurrently: '^4.1.0',
      'cross-spawn': '^6.0.5',
      eslint: '^5.16.0',
      'eslint-config-standard': '^12.0.0',
      'eslint-plugin-flowtype': '2.50.1',
      'eslint-plugin-import': '2.16.0',
      'eslint-plugin-node': '^9.1.0',
      'eslint-plugin-prettier': '^3.1.0',
      'eslint-plugin-promise': '^4.1.1',
      'eslint-plugin-standard': '^4.0.0',
      'jest-theories': '^1.2.0',
      nodemon: '^1.19.1',
      prettier: '^1.17.1'
    }
  },
  fs: { numberOfFiles: 49 },
  debug: [
    '2020-08-28T09:30:26.213Z angular/cli config Angular CLI not found.\n',
    '2020-08-28T09:30:26.619Z jest/config Detected Jest.\n',
    '2020-08-28T09:30:26.620Z jest/config Configured Jest.\n',
    '2020-08-28T09:30:26.623Z project Wallaby Node version: v12.9.0\n',
    '2020-08-28T09:30:26.623Z project Wallaby config: C:\\personal\\air-monitor-reporting\\auto.detect\n',
    '2020-08-28T09:30:28.403Z project File cache: C:\\Users\\josh\\.vscode\\extensions\\wallabyjs.wallaby-vscode-1.0.234\\projects\\ce1d08d38d65f482\n',
    '2020-08-28T09:30:28.535Z uiService Listening port 51235\n',
    '2020-08-28T09:30:28.588Z workers Parallelism for initial run: 6, for regular run: 3\n',
    '2020-08-28T09:30:28.588Z workers Starting run worker instance #0\n',
    '2020-08-28T09:30:28.588Z workers Starting run worker instance #1\n',
    '2020-08-28T09:30:28.589Z workers Starting run worker instance #2\n',
    '2020-08-28T09:30:28.589Z workers Starting run worker instance #3\n',
    '2020-08-28T09:30:28.589Z workers Starting run worker instance #4\n',
    '2020-08-28T09:30:28.589Z workers Starting run worker instance #5\n',
    '2020-08-28T09:30:28.590Z workers Web server is listening at 55013\n',
    '2020-08-28T09:30:28.591Z project File cache requires some updates, waiting required files from IDE\n',
    '2020-08-28T09:30:28.880Z project Stopping process pool\n',
    '2020-08-28T09:30:28.883Z project Test run started; run priority: 3\n',
    '2020-08-28T09:30:28.895Z project Running all tests\n',
    '2020-08-28T09:30:28.902Z workers Starting test run, priority: 3\n',
    '2020-08-28T09:30:28.902Z workers Distributing tests between 6 workers\n',
    '2020-08-28T09:30:28.903Z workers Running tests in parallel\n',
    '2020-08-28T09:30:28.904Z nodeRunner Starting sandbox [worker #0, session #gtob0]\n',
    '2020-08-28T09:30:28.904Z nodeRunner Starting sandbox [worker #1, session #l5qmu]\n',
    '2020-08-28T09:30:28.904Z nodeRunner Starting sandbox [worker #2, session #06um6]\n',
    '2020-08-28T09:30:28.904Z nodeRunner Starting sandbox [worker #3, session #gqv9l]\n',
    '2020-08-28T09:30:28.904Z nodeRunner Starting sandbox [worker #4, session #5ugh3]\n',
    '2020-08-28T09:30:28.904Z nodeRunner Starting sandbox [worker #5, session #3mrpl]\n',
    '2020-08-28T09:30:28.904Z nodeRunner Preparing sandbox [worker #0, session #gtob0]\n',
    '2020-08-28T09:30:28.904Z nodeRunner Preparing sandbox [worker #1, session #l5qmu]\n',
    '2020-08-28T09:30:28.904Z nodeRunner Preparing sandbox [worker #2, session #06um6]\n',
    '2020-08-28T09:30:28.905Z nodeRunner Preparing sandbox [worker #3, session #gqv9l]\n',
    '2020-08-28T09:30:28.905Z nodeRunner Preparing sandbox [worker #4, session #5ugh3]\n',
    '2020-08-28T09:30:28.905Z nodeRunner Preparing sandbox [worker #5, session #3mrpl]\n',
    '2020-08-28T09:30:28.985Z workers Started run worker instance (delayed) #0\n',
    '2020-08-28T09:30:28.985Z nodeRunner Prepared sandbox [worker #0, session #gtob0]\n',
    '2020-08-28T09:30:28.986Z workers [worker #0, session #gtob0] Running tests in sandbox\n',
    '2020-08-28T09:30:29.027Z workers Started run worker instance (delayed) #1\n',
    '2020-08-28T09:30:29.028Z nodeRunner Prepared sandbox [worker #1, session #l5qmu]\n',
    '2020-08-28T09:30:29.028Z workers [worker #1, session #l5qmu] Running tests in sandbox\n',
    '2020-08-28T09:30:29.081Z workers Started run worker instance (delayed) #2\n',
    '2020-08-28T09:30:29.081Z nodeRunner Prepared sandbox [worker #2, session #06um6]\n',
    '2020-08-28T09:30:29.081Z workers [worker #2, session #06um6] Running tests in sandbox\n',
    '2020-08-28T09:30:29.085Z workers Started run worker instance (delayed) #3\n',
    '2020-08-28T09:30:29.085Z nodeRunner Prepared sandbox [worker #3, session #gqv9l]\n',
    '2020-08-28T09:30:29.085Z workers [worker #3, session #gqv9l] Running tests in sandbox\n',
    '2020-08-28T09:30:29.102Z workers Started run worker instance (delayed) #5\n',
    '2020-08-28T09:30:29.102Z nodeRunner Prepared sandbox [worker #5, session #3mrpl]\n',
    '2020-08-28T09:30:29.102Z workers [worker #5, session #3mrpl] Running tests in sandbox\n',
    '2020-08-28T09:30:29.141Z workers Started run worker instance (delayed) #4\n',
    '2020-08-28T09:30:29.141Z nodeRunner Prepared sandbox [worker #4, session #5ugh3]\n',
    '2020-08-28T09:30:29.141Z workers [worker #4, session #5ugh3] Running tests in sandbox\n',
    '2020-08-28T09:30:40.116Z workers [l5qmu] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:40.118Z workers [l5qmu] Test executed: total should be 0.01 when given these samples: [{"fields":100,"fullFibers":1,"halfFibers":0,"type":"BLANK"},{"fields":100,"fullFibers":1,"halfFibers":0,"type":"BLANK"}]\n',
    '2020-08-28T09:30:40.120Z workers [l5qmu] Test executed: total should be 0.01 when given these samples: [{"fields":100,"fullFibers":1,"halfFibers":0,"type":"BLANK"},{"fields":100,"fullFibers":1,"halfFibers":0,"type":"BLANK"},{"fields":100,"fullFibers":1,"halfFibers":0,"type":"STANDARD"}]\n',
    '2020-08-28T09:30:40.129Z workers [l5qmu] Run 2 test(s), skipped 0 test(s)\n',
    '2020-08-28T09:30:40.132Z workers [l5qmu] Sandbox is responsive, closing it\n',
    '2020-08-28T09:30:40.161Z workers [5ugh3] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:40.162Z workers [5ugh3] Test executed: should be 54.46 when given sample: {"fullFibers":35,"halfFibers":20,"fields":100}, undefined fields & 0.00785 filterArea\n',
    '2020-08-28T09:30:40.180Z workers [5ugh3] Run 1 test(s), skipped 0 test(s)\n',
    '2020-08-28T09:30:40.182Z workers [5ugh3] Sandbox is responsive, closing it\n',
    '2020-08-28T09:30:40.527Z workers [3mrpl] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:40.530Z workers [3mrpl] Test executed: should be 422.8 when given sample: {"timeOn":"08:43","timeOff":"09:39","lpmOn":7.5,"lpmOff":7.6}\n',
    '2020-08-28T09:30:40.534Z workers [3mrpl] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:40.535Z workers [3mrpl] Test executed: total should be 1.5 when given [{"fullFibers":1,"halfFibers":1,"type":"BLANK"}]\n',
    '2020-08-28T09:30:40.535Z workers [3mrpl] Test executed: total should be 1.5 when given [{"fullFibers":1,"halfFibers":1,"type":"BLANK"},{"fullFibers":1,"halfFibers":1,"type":"STANDARD"}]\n',
    '2020-08-28T09:30:40.535Z workers [3mrpl] Test executed: total should be 0 when given [{"fullFibers":1,"halfFibers":1,"type":"STANDARD"}]\n',
    '2020-08-28T09:30:40.542Z workers [3mrpl] Run 4 test(s), skipped 0 test(s)\n',
    '2020-08-28T09:30:40.544Z workers [3mrpl] Sandbox is responsive, closing it\n',
    '2020-08-28T09:30:40.586Z workers [06um6] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:40.598Z workers [06um6] Test executed: should be 7.55 when lpmOn: 7.5 & lpmOff: 7.6\n',
    '2020-08-28T09:30:40.599Z workers [gqv9l] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:40.600Z workers [06um6] Test executed: should be 3.8 when lpmOn:  & lpmOff: 7.6\n',
    '2020-08-28T09:30:40.600Z workers [gqv9l] Test executed: total should be 56 when timeOn: 08:43 & timeOff: 09:39\n',
    '2020-08-28T09:30:40.601Z workers [gqv9l] Test executed: total should be 720 when timeOn: 10:40 & timeOff: 22:40\n',
    '2020-08-28T09:30:40.604Z workers [06um6] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:40.606Z workers [06um6] Test executed: total should be 1.5 when given [{"fullFibers":1,"halfFibers":1,"type":"BLANK"}]\n',
    '2020-08-28T09:30:40.607Z workers [gqv9l] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:40.607Z workers [06um6] Test executed: total should be 3 when given [{"fullFibers":1,"halfFibers":1,"type":"BLANK"},{"fullFibers":1,"halfFibers":1,"type":"STANDARD"}]\n',
    '2020-08-28T09:30:40.608Z workers [gqv9l] Test executed: given 100 full fibers & 100 half fibers, output should be 150\n',
    '2020-08-28T09:30:40.609Z workers [06um6] Test executed: total should be 1.5 when given [{"fullFibers":1,"halfFibers":1,"type":"STANDARD"}]\n',
    '2020-08-28T09:30:40.609Z workers [gqv9l] Test executed: given 100 full fibers & 100 half fibers, output should be 150\n',
    '2020-08-28T09:30:40.610Z workers [gqv9l] Test executed: given 100 full fibers & 500 half fibers, output should be 350\n',
    '2020-08-28T09:30:40.610Z workers [gqv9l] Test executed: given 1 full fibers & 1 half fibers, output should be 1.5\n',
    '2020-08-28T09:30:40.618Z workers [06um6] Run 5 test(s), skipped 0 test(s)\n',
    '2020-08-28T09:30:40.619Z workers [gqv9l] Run 6 test(s), skipped 0 test(s)\n',
    '2020-08-28T09:30:40.620Z workers [06um6] Sandbox is responsive, closing it\n',
    '2020-08-28T09:30:40.621Z workers [gqv9l] Sandbox is responsive, closing it\n',
    '2020-08-28T09:30:45.285Z workers [gtob0] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:45.286Z workers [gtob0] Test executed: total should be 1.75 when given these samples: [{"fullFibers":1,"halfFibers":1,"type":"BLANK"},{"fullFibers":2,"halfFibers":0,"type":"BLANK"}]\n',
    '2020-08-28T09:30:45.287Z workers [gtob0] Test executed: total should be 1.75 when given these samples: [{"fullFibers":1,"halfFibers":1,"type":"BLANK"},{"fullFibers":2,"halfFibers":0,"type":"BLANK"},{"fullFibers":2,"halfFibers":0,"type":"STANDARD"}]\n',
    '2020-08-28T09:30:45.291Z workers [gtob0] Loaded unknown number of test(s)\n',
    '2020-08-28T09:30:45.292Z workers [gtob0] Test executed: renders without crashing\n',
    '2020-08-28T09:30:45.299Z workers [gtob0] Run 3 test(s), skipped 0 test(s)\n',
    '2020-08-28T09:30:45.300Z workers [gtob0] Sandbox is responsive, closing it\n',
    '2020-08-28T09:30:45.301Z workers Merging parallel test run results\n',
    '2020-08-28T09:30:45.303Z project Test run finished\n',
    '2020-08-28T09:30:45.304Z project Processed console.log entries\n',
    '2020-08-28T09:30:45.304Z project Processed loading sequences\n',
    '2020-08-28T09:30:45.305Z project Test name duplicate: given 100 full fibers & 100 half fibers, output should be 150\n',
    '2020-08-28T09:30:45.305Z project Processed executed tests\n',
    '2020-08-28T09:30:45.307Z project Processed code coverage\n',
    '2020-08-28T09:30:45.379Z project Test run result processed and sent to IDE\n'
  ]
}
```

**Comment by NikGovorov** [comment]
Hi @JoshWhite,

We can not reproduce the issue. Could you please execute the following command in terminal `code --list-extensions` and share the output with us?

**Comment by JoshWhite** [comment]
Sure:

```
$ code --list-extensions
alefragnani.rtf
asvetliakov.snapshot-tools   
bajdzis.vscode-database      
dbaeumer.vscode-eslint       
eamodio.gitlens
ee92.folderize
formulahendry.auto-rename-tag
lucasazzola.vscode-csproj
mikestead.dotenv
ms-azuretools.vscode-docker
ms-dotnettools.csharp
lucasazzola.vscode-csproj
mikestead.dotenv
ms-azuretools.vscode-docker
ms-dotnettools.csharp
ms-vscode-remote.remote-wsl
ms-vscode.cpptools
ms-vsliveshare.vsliveshare
ms-vsts.team
msjsdiag.debugger-for-chrome
purplestone.savebackup
Shan.code-settings-sync
sterin.msbuild-tools
streetsidesoftware.code-spell-checker
Trezm.type-doc-vscode
vscode-icons-team.vscode-icons
WallabyJs.wallaby-vscode
wangtao0101.vscode-js-import
wayou.vscode-todo-highlight

```
Although currently, the following are disabled:
```
ms-dotnettools.csharp
msjsdiag.debugger-for-chrome
ms-vscode-remote.remote-wsl
Trezm.type-doc-vscode
lucasazzola.vscode-csproj
wangtao0101.vscode-js-import
```

**Comment by NikGovorov** [comment]
Could you please open Developer Tools, try to reproduce the issue and see if there are any errors reported to the console?

![ScreenClip](https://user-images.githubusercontent.com/129254/91675937-72f21e00-eb81-11ea-9a31-b5ac0bbb4fd9.png)
![ScreenClip](https://user-images.githubusercontent.com/129254/91675944-77b6d200-eb81-11ea-9aa4-2ed316b34d25.png)


Another thing I would like you to do is to disable all installed extensions, enable Wallaby.js extension and try to reproduce the issue with only Wallaby.js extension enabled?

![ScreenClip](https://user-images.githubusercontent.com/129254/91676041-d714e200-eb81-11ea-9332-104fcb5035ff.png)

**Comment by JoshWhite** [comment]
Hey, 

Here's a screen shot of from my console (text output provided below).
The console entries appear as soon as I load VSCode. Nothing changes if I replicate the bug. 

![image](https://user-images.githubusercontent.com/1931725/91821685-a343c600-ec2e-11ea-9964-de93d6f1a177.png)

```
console.ts:137 [Extension Host] (node:22540) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
t.log @ console.ts:137
$logExtensionHostMessage @ mainThreadConsole.ts:39
_doInvokeHandler @ rpcProtocol.ts:409
_invokeHandler @ rpcProtocol.ts:394
_receiveRequest @ rpcProtocol.ts:310
_receiveOneMessage @ rpcProtocol.ts:237
(anonymous) @ rpcProtocol.ts:105
fire @ event.ts:587
fire @ ipc.net.ts:463
_receiveMessage @ ipc.net.ts:747
(anonymous) @ ipc.net.ts:602
fire @ event.ts:587
acceptChunk @ ipc.net.ts:240
(anonymous) @ ipc.net.ts:201
t @ ipc.net.ts:29
emit @ events.js:203
addChunk @ _stream_readable.js:295
readableAddChunk @ _stream_readable.js:276
Readable.push @ _stream_readable.js:210
onStreamRead @ internal/stream_base_commons.js:166
console.ts:137 [Extension Host] [Wallaby Console] VS Code extension v1.0.234
console.ts:137 [Extension Host] [Wallaby Console] Core v1.0.947
console.ts:137 [Extension Host] Failed to connect to provider "Command failed with exit code 1: git config --local --get-regexp ^remote.*.url"
console.ts:137 [Extension Host] [vscode-icons] v10.2.0 activated!
```

**Comment by JoshWhite** [comment]
I've just tried this and I still get the same result.

> Another thing I would like you to do is to disable all installed extensions, enable Wallaby.js extension and try to reproduce the issue with only Wallaby.js extension enabled?

Not sure if this will help, but here's a before and after of the elements panel:

## Before
![image](https://user-images.githubusercontent.com/1931725/91822851-4517e280-ec30-11ea-9805-e77319e6042d.png)

## After

![image](https://user-images.githubusercontent.com/1931725/91822873-4c3ef080-ec30-11ea-92b2-fe143354b3e5.png)

The markup has obviously gone awry, but I've no idea why...

**Comment by smcenlly** [comment]
Thanks for the update. We've isolated the problem and we're working on a fix. Interestingly, it looks like this problem isn't reproducible in `vscode-insiders` and is also specific to Windows.

We'll provide an update when we have a fix ready for you.

**Comment by NikGovorov** [comment]
Thanks for reporting the issue. It is fixed and the fix is published in 1.0.235.

**Comment by JoshWhite** [comment]
I can confirm this works after downloading the latest version. Thanks!
